frontend/src/core/api/types.js

```diff
@@ -14,6 +14,21 @@ export type EntityTranslation = {|
 |};
 
 
+/**
+ * Comments pertaining to a translation.
+ */
+export type TranslationComment = {|
+    +author: string,
+    +author_id: number,
+    +username: string,
+    +user_gravatar_url_small: string,
+    +timestamp: string,
+    +date_iso: string,
+    +content: string,
+    +translation: number,
```

Comment:
(Hah, I now realize that you haven't implemented the `api` yet :P So please do change this type, and when you add the api method, please make sure to map the data to these names. )

Reply:
BTW, this patch only implements translation comments, which will be loaded as part of the existing `getHistory()` API call. Team comments (the ones in the sidebar) will be implemented in a followup patch (and will require a new API call).

Since we'll need this type for team comments as well, I think it's fine to keep it here (as opposed to history reducer), but I'd rename it to just `Comment`.

Reply:
Renaming from `TranslationComment` to `Comment` in `types` causes conflicts do to the `Comment` component.  Maybe something like `CommentDetails` could work??

Reply:
I'd go for `CommentType`, I think. I remember resolving such a conflict by calling the type `DbThing` instead of `Thing` but that wasn't great and I have been able to rename that recently. 

Note though that this type here has a `translation` key that will **not** be in the data for other types of comments. So maybe it's just better to keep this as `TranslationComment` for now, and refactor later on?